test/QCefViewTest/customcefview.cpp

```diff
@@ -29,7 +50,9 @@ CustomCefView::onQCefUrlRequest(const QString& url)
                          "Url: %1")
                    .arg(url);
 
-  QMessageBox::information(this->window(), title, text);
+  dispatchToMainThread([=]() {
```

Comment:
Hi @sketch34 , can you simply replace this method call with QmetaObject::invokeMethod like the flowing code
`  QMetaObject::invokeMethod(
    this,
    [=]() {
      QString title("QCef Query Request");
      QString text = QString("Current Thread: QT_UI\r\n"
                             "Query: %1")
                       .arg(query.reqeust());

      QMessageBox::information(this->window(), title, text);

      QString response = query.reqeust().toUpper();
      query.setResponseResult(true, response);
      responseQCefQuery(query);
    },
    Qt::QueuedConnection);`